fvwm/fvwm3.desktop.in

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=FVWM3
+Comment=The fvwm3 window manager
+Type=XSession
+Exec=@bindir@/fvwm3
+TryExec=@bindir@/fvwm3
```

Comment:
As I understand it, `Exec` is for absolute paths. and if you use `TryExec` without a path, as in:

```
TryExec=fvwm3
```

Then if the `Exec` fails due to `$(bindir)/fvwm3` not being found disk, then at least we'd have $PATH to fall back on.

Reply:
Also interesting. Then my suggestion would be to remove `TryExec` completely. I don't think relying on `$PATH` is ever a good idea for security reasons. What do you think?

Reply:
Sure, that works for me.  At this point, beyond someone manually moving the binary out of the way, we're guaranteeing the file to be in the correct location at the point this file is generated anyway.

Reply:
BTW, just for reference, I compared it to this:
https://invent.kde.org/plasma/plasma-workspace/-/blob/master/login-sessions/plasmax11.desktop.cmake

Yes, that's what KDE installs. Both `Type=XSession` and `TryExec=<fullpath>` in there :o

But of course I want to do it *correctly*. Give me a while to update this, I first want to make sure my sddm discovers and launches fvwm3 correctly with these changes :)